konfera/models/ticket.py

```diff
@@ -33,13 +33,15 @@ def __str__(self):
             last_name=self.last_name
         ).strip()
 
+    def discount_calculator(self):
+        if self.discount_code:
+            return self.type.price * self.discount_code.discount / 100
+        return 0
```

Comment:
If you can avoid have multiple return in function, please do that in future. It will improve readability and reduce code complexity. You can do it like this:

```
def discount_calculator(self):
       discount = 0

       if self.discount_code:
           discount = self.type.price * self.discount_code.discount / 100
       return discount
```